template/cordova/lib/run.js

```diff
@@ -169,20 +135,10 @@ module.exports.help = function () {
     process.exit(0);
 };
 
-
-function getWindowsTargetVersion() {
```

Comment:
This is not required anymore as we rely on build - we run the first build target.

var buildTargets = build.getBuildTargets(args.win, args.phone, args.appx);

..
+     if (!buildTargets || buildTargets.lenght <= 0) {
+         return Q.reject('Unable to determine deploy target.');
+     }
+
+     // we deploy the first build target so we use buildTargets[0] to determine
+     // what project type we should deploy
+     var projectType = projFileToType(buildTargets[0]);